src/main/java/TicketService.java

```diff
@@ -62,8 +61,12 @@ public static void main(String[] args) {
         List<BusTicket> busTickets = fileWork.readFromFile("src/main/resources/tickets.json");
         TicketService.printTicketsInfo(busTickets);
 
-        TicketDataBaseDao ticketDataBaseDao = new TicketDataBaseDaoImpl();
-        UserDataBaseDao userDataBaseDao = new UserDataBaseDaoImpl(ticketDataBaseDao);
+        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(AppConfig.class);
```

Comment:
use abstraction here please - ApplicationContext

Reply:
Thank you ) Good day !!